safe-global/web-core
# feat: Add spending limit tx flow

## What it solves

- Adds spending limit tx flow
- Formats `spent` amount in spending limit table
- Updates spending limit data when the tx history updates as opposed to updating when there was a SUCCESS event
- Extracts `getTxOptions` from `SignOrExecuteForm`

## How to test

1. Create a new tx for a token that has a spending limit
2. Observe gas being estimated
3. Execute the tx
4. Observe tx being executed
